service/src/main/java/com/epam/rft/atsy/service/impl/ApplicationsServiceImpl.java

```diff
@@ -67,7 +70,16 @@
 
     List<ApplicationEntity> applicationEntities = pageResult.getContent();
```

Comment:
Maybe this local variable is not important. There is only 1 call, you can use "pageResult.getContent()" there.